src/test/java/junit/samples/ListTest.java

```diff
@@ -11,60 +11,58 @@
  * A sample test case, testing {@link java.util.ArrayList}.
  */
 public class ListTest extends TestCase {
-    protected List<Integer> fEmpty;
-    protected List<Integer> fFull;
+    private List<Integer> emptyList;
+
+    private List<Integer> fullList;
 
-    public static void main(String[] args) {
-        junit.textui.TestRunner.run(suite());
-    }
 
     @Override
     protected void setUp() {
-        fEmpty = new ArrayList<Integer>();
-        fFull = new ArrayList<Integer>();
-        fFull.add(1);
-        fFull.add(2);
-        fFull.add(3);
+        emptyList = new ArrayList<Integer>();
+        fullList = new ArrayList<Integer>();
+        fullList.add(1);
+        fullList.add(2);
+        fullList.add(3);
     }
 
     public static Test suite() {
         return new TestSuite(ListTest.class);
     }
 
     public void testCapacity() {
-        int size = fFull.size();
+        int size = fullList.size();
         for (int i = 0; i < 100; i++) {
-            fFull.add(i);
+            fullList.add(i);
         }
-        assertTrue(fFull.size() == 100 + size);
+        assertTrue(fullList.size() == 100 + size);
     }
 
     public void testContains() {
-        assertTrue(fFull.contains(1));
-        assertTrue(!fEmpty.contains(1));
+        assertTrue(fullList.contains(1));
+        assertTrue(!emptyList.contains(1));
     }
 
     public void testElementAt() {
-        int i = fFull.get(0);
+        int i = fullList.get(0);
         assertTrue(i == 1);
```

Comment:
Do you think we should use `assertEquals()` (with as useful message) instead of `assertTrue()` so you'll get a useful stack trace if this fails? Future users might look at this file as an example of how to write a good JUnit test.